### ISSUE TITLE
undefined method `script_name' for nil:NilClass

### ISSUE THREAD

**Issue by yuta17** [k0kubun/rspec-openapi#19]
Hi. When I type `RAILS_ENV=test OPENAPI=1 bundle exec rspec [spec file path]` the following error happens.

```ruby
  NoMethodError:
    undefined method `script_name' for nil:NilClass
  # /usr/local/bundle/gems/rspec-openapi-0.3.12/lib/rspec/openapi/record_builder.rb:64:in `find_rails_route'
  # /usr/local/bundle/gems/rspec-openapi-0.3.12/lib/rspec/openapi/record_builder.rb:19:in `build'
  # /usr/local/bundle/gems/rspec-openapi-0.3.12/lib/rspec/openapi/hooks.rb:12:in `block in <top (required)>'
```

The following is the content of the target test file.

```ruby
require 'rails_helper'

RSpec.describe V2::Admin::NotificationsController, type: :request do
  describe 'GET /v2/admin/notifications' do
    subject { get v2_admin_notifications_url, params: params }
    let!(:users) { create_list(:user) }
    let(:params) { nil }

    it { is_expected.to be 200 }
  end
end
```
I found that  if I do not use `create_list`, no error occurs.

gem version: 0.3.12

**Comment by k0kubun** [comment]
What is `create_list`? I don't think it's part of rspec-openapi. Could you prepare a repository that reproduces your issue? Otherwise I can't debug `create_list`.

**Comment by k0kubun** [comment]
Although I couldn't test your `create_list` case, I could figure out a case where the same backtrace happens. I'll release a fix as v0.3.13.
